### ISSUE TITLE
AWS cost attribution (unplanned addition): handle "Not found" error showing up

### ISSUE THREAD

**Issue by consideRatio** [2i2c-org/infrastructure#4948]
This "Not found" warning shows up along with a browser web console error.

![Image](https://github.com/user-attachments/assets/b76ef33c-c3eb-4ca5-b425-0b4f1ebf0f29)

```
backend_srv.ts:132 
GET https://grafana.openscapes.2i2c.cloud/api/prometheus/grafana/api/v1/rules?dashboard_uid=null 404 (Not Found)
```

I suspect it relates to deploying the cost attribution dashboard based on a .jsonnet dashboard definition that isn't adjusting to the specific grafana instance or similar.

## Definition of done

- We no longer get these errors in our AWS grafana instances.

**Comment by consideRatio** [comment]
It seems this is https://github.com/grafana/grafana/issues/87891, and was triggered by #4923, and will be resolved by #4684 that bumps grafana to a version that has it fixed.

**Comment by consideRatio** [comment]
Verified to be resolved!

Thank you @sgibson91 for getting support chart bumped, and thank you @yuvipanda for working a system to help us do routine maintenance like updating grafana!
